AceComm-3.0/ChatThrottleLib.lua

```diff
@@ -251,6 +251,14 @@ function ChatThrottleLib:Init()
 		end)
 	end
 
+	-- v27: Hook BNSendGameData for traffic logging
```

Comment:
nit: v29

Reply:
Fixed; will squash it down once you're happy it's good to go.